src/common/types/store-data/needs-review-scan-result-data.ts

```diff
@@ -12,12 +12,12 @@ import {
 } from 'common/types/store-data/unified-data-interface';
 
 export interface NeedsReviewScanResultStoreData {
-    results: UnifiedResult[];
-    rules: UnifiedRule[];
-    platformInfo?: PlatformData;
-    toolInfo?: ToolData;
-    targetAppInfo?: TargetAppData;
-    timestamp?: string;
-    scanIncompleteWarnings?: ScanIncompleteWarningId[];
-    screenshotData?: ScreenshotData;
+    results: UnifiedResult[] | null;
+    rules: UnifiedRule[] | null;
+    platformInfo?: PlatformData | null;
```

Comment:
It reads a little strange to have `thing?: Type | null` but I still like `?` for optional fields...

Reply:
Agreed!